app/views/users/edit.html.haml

```diff
@@ -1,145 +1,44 @@
-:css
-  .required {
-      color: black;
-  }
-  table.form {
-    width: auto;
-  }
-  table.form td {
-    width: 500px;
-  }
-
-:javascript
-  /* refer to input fields by name for rules */
-  /* refer to input fields by id for equalTo */
-  jQuery(document).ready(function() {
```

Comment:
you removed all this JS code without replacing it?

Reply:
This file is still in progress